cmd/minishift/cmd/services/services.go

```diff
@@ -27,5 +27,4 @@ var ServicesCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cmd.Help()
 	},
-	Hidden: true,
```

Comment:
@anjannath so now we are exposing this command to users?

Reply:
Yes

Reply:
did we expose the Services namespace with a description... I remember I add one, but is it shown?

Reply:
@gbraad It shows the description